src/SolutionAssemblyInfo.cs

```diff
@@ -1,38 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
You mean GitVersion actually modifies source files tracked by source control rather than adding a generated file to the build?